src/proto/project.proto

```diff
@@ -51,6 +57,7 @@ message Project {
   uint64 created_at = 8;             // Time of project creation (UTC seconds epoch)
   uint64 updated_at = 9;             // Time of last project update (UTC seconds epoch)
   repeated ProjectAsset assets = 10; // The listing of all assets associated with the project
+  optional ProjectType project_type = 11; // The type of project
```

Comment:
Could name as just `type` here and in the Create and Update requests since is already in the context of Project

Reply:
Yeah, definitely makes sense. I'll update this.

Reply:
Oh I meant keeping ProjectType but changing `project_type` to `type`

Reply:
oooh got it :)